src/services/tag.service.ts

```diff
@@ -34,16 +37,26 @@ export class TagService {
           updatedAt: new Date().toString(),
         }) as Promise<Tag>;
       } else {
-        const oneDay: number = 60 * 60 * 24 * 1000;
-        const isOneDay: boolean =
-          new Date().getTime() - new Date(foundTag.updatedAt).getTime() >
-          oneDay;
+        const oneDay = day;
+        const isToday = today(foundTag.updatedAt) > oneDay;
```

Comment:
move this function to DateUtils,
this line should be like this

`const isToday = today(foundTag.updatedAt);`